src/components/input/input.css

```diff
@@ -1,16 +1,15 @@
 @import '../../styles/colors-v2.css';
 @import "../../styles/spacing.css";
-@import "../../styles/typography.css";
+@import "../../styles/typography-v2.css";
 
 .input {
   border: 1px solid var(--mds-grey-12);
   border-radius: 3px;
   box-sizing: border-box;
-  font-family: var(--mavenlink-type-font-family); /* stylelint-disable-line mds/typography */
-  font-size: var(--mavenlink-type-base-size); /* stylelint-disable-line mds/typography */
+  font: var(--mds-type-content);
   height: 32px;
   margin: 0;
-  padding: 6px var(--spacing-x-large) 7px var(--spacing-medium);
+  padding: 0 var(--spacing-x-large) 0 var(--spacing-medium);
```

Comment:
I wonder why we had such specific top and bottom padding...

Reply:
We basically don't want to specify top or bottom. We want the text to just be vertically aligned (defaults on inputs) and take up as much space as specified in the font.

Reply:
This is just the ramification of choosing the short-hand property instead of individual properties like `padding-left` and `padding-right`.

Reply:
Sorry, I was wondering out loud why the previous values were so specifically set. 😄

Reply:
Ah, yeah. That.

I... don't recall. Part of me wants to say that it was originally intended for the inputs to be 32px high but we didn't want to set height? Like we wanted the component to "organically" take up its own space from its sub-dimensions. The 6 vs 7 difference I think was a trick to get the font to look more centered.